src/python/pants/backend/python/subsystems/twine.py

```diff
@@ -60,6 +63,17 @@ def register_options(cls, register):
                 "non-standard location."
             ),
         )
+        register(
+            "--ca-certs-path",
+            advanced=True,
+            type=str,
+            default="<inherit>",
+            help=(
+                "Path to a file containing PEM-format CA certificates used for verifying secure "
+                "connections when publishing python distributions.\n\n"
+                "Uses the value from `[GLOBAL].ca_certs_path` by default."
```

Comment:
How do you disable the feature? Note that you can't set `None` in TOML :/ https://github.com/pantsbuild/pants/issues/11719

We've been using `"<none>"` as a bad workaround

Reply:
Ah, that's a good question. I think setting it to the empty string would work. Perhaps worth mentioning in the help text?

Reply:
Right, yes, empty string disables this: https://github.com/pantsbuild/pants/pull/13593/files#diff-6b3bf4da6a8548617d0fb710a3a496830fea7a584e075f8fdee6814fa0b43590R104

Reply:
It technically would because of using `not ca_certs_path` instead of `ca_certs_paths is None`. But that's pretty intricate. I think I recommend using `<none>` instead - lots of precedent for that

A new paragraph explaining <inherit> (the default) uses global option, and `<none>` disables entirely

If you do go with `<none>`, note that you must use backticks when documenting it! Otherwise markdown won't render it